client/src/store/admin/api.ts

```diff
@@ -62,70 +33,13 @@ export const api = createApi({
   tagTypes: apiObjectNames,
   // REMEMBER: our api needs trailing slashes
   endpoints: (builder) => {
-    const getCreator = <R>(apiObjectName: ApiObjectName) =>
```

Comment:
I thought I would end up using these factory functions in the new store, so I factored them into a separate file.  This didn't end up being required, but I liked it separate anyway, so I kept in the refactor